jakarta-eclipse/org.eclipse.lsp4jakarta.core/src/org/jakarta/jdt/jax_rs/ResourceMethodMultipleEntityParamsQuickFix.java

```diff
@@ -0,0 +1,105 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation, Bera Sogut and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation, Bera Sogut - initial API and implementation
+ *******************************************************************************/
+
+package org.jakarta.jdt.jax_rs;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.jdt.core.dom.ASTNode;
+import org.eclipse.jdt.core.dom.Annotation;
+import org.eclipse.jdt.core.dom.IExtendedModifier;
+import org.eclipse.jdt.core.dom.IMethodBinding;
+import org.eclipse.jdt.core.dom.MethodDeclaration;
+import org.eclipse.jdt.core.dom.Name;
+import org.eclipse.jdt.core.dom.SingleVariableDeclaration;
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.Diagnostic;
+import org.jakarta.codeAction.IJavaCodeActionParticipant;
+import org.jakarta.codeAction.JavaCodeActionContext;
+import org.jakarta.codeAction.proposal.ChangeCorrectionProposal;
+import org.jakarta.codeAction.proposal.RemoveEntityParamsProposal;
+
+/**
+ * Quick fix for the ResourceMethodMultipleEntityParams diagnostic in
+ * ResourceMethodDiagnosticsCollector.
+ *
+ * @author Bera Sogut
```

Comment:
Consider clarifying what this quick fix does in this comment, ie. "Keep all but one entity parameter"